code/go/0chain.net/smartcontract/zcnsc/mint.go

```diff
@@ -49,6 +49,12 @@ func (zcn *ZCNSmartContract) Mint(trans *transaction.Transaction, inputData []by
 	}
 
 	numAuth, err := getAuthorizerCount(ctx)
+	if err != nil {
```

Comment:
The err is nil even the numAuth does not exist. So we should also check if numAuth is 0.

Reply:
I don't think we should check it, since we should invalidate any number of signatures in case we have 0 authorizers. In general, it should not differ how much authorizers we do ave 0, 1 or k, it should work the same way

Reply:
> since we should invalidate any number of signatures. in case we have 0 authorizers. 

Yes, that's what I mean, `numAuth` is the number of authorizers, and we should stop the Mint if it is 0, otherwise the zero division panic would happen as we have encountered.

Reply:
Actually, the panic does not happen in the unit test case that I have added in this PR. Will check it again if I have written the unit test correctly.

Reply:
@areebusmaniblr yes, that should work to fix the panic (thought this branch would never panic as it does not have the latest code from staging), but we'd better check the 0 number of authorizer explicitly instead of letting the empty sliced signatures' verification code to do the 0 checking, in any chance if the `verifySignature` does not do the empty checking, we will panic here https://github.com/0chain/0chain/blob/59c0b3a1481d8e3c776c612987f494270d256e61/code/go/0chain.net/smartcontract/zcnsc/mint.go#L121. Anyway, I'll do a quick commit and merge this PR.